examples/forms.amp.html

```diff
@@ -72,7 +86,8 @@
 <form method="post"
       action="/form/html/post"
       action-xhr="/form/echo-json/post"
-      target="_blank">
+      target="_blank"
+      on="submit:lightbox1">
```

Comment:
So, is this a "onsubmit" like in the browser? Or logically a "submit success"? I.e. do we wait for submit to finish?